recipe/meta.yaml

```diff
@@ -19,10 +19,10 @@ requirements:
     - python
     - pip
   run:
-    - python >=2.7.0
+    - python
     - requests >=2.0.0,<3
     - pyjwt >=1.0.0,<3
-    - cryptography >=0.6,<4
+    - cryptography >=0.6,<38
```

Comment:
https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/8573a1679fee629c0327f2fe94608dadbdd4e48c/setup.py#L74-L87